### PR TITLE
Fix: Do not render icon before the first breadcrumb item #JALL-32

### PR DESCRIPTION
**Pull request by dlouhak**

### CODE REVIEW

File: packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcru
                 </li>
               )}
               <li className="d-none d-tablet-flex">
-                <Icon name="chevron-right" />
+                {index !== 0 && <Icon name="chevron-right" />}
```
Comment (literat) [inline packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx:43]: Could we have a test for this, please?

Comment (literat) [reply packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx:43]: Something like "find all `li` in `ul` or `nav` a and the first item's content is not the SVG".

File: packages/web-twig/src/Resources/components/Breadcrumbs/index.twig [hunk 2]
```diff
@@ -23,7 +23,9 @@
                     </li>
                 {% endif %}
                 <li class="d-none d-tablet-flex">
-                    <Icon name="chevron-right" />
+                    {% if loop.index != 1 %}
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Breadcrumbs/index.twig:26]: If you would you `index0` then the condition could be basically the same as in JS. But 🤷.

**Review by crishpeen — APPROVED** [review]

**Review by adamkudrna — APPROVED** [review]

**Review by tomassychra — APPROVED** [review]

File: packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx (hunk 1 above)
Comment (literat) [reply packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx:43]: @dlouhak I do not see any test or any answer...

Comment (dlouhak) [reply packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx:43]: You're right, but then we should make tests for another test cases: presence of the element with `goBackTitle` on mobile screens, different visual and a11y properties for last breadcrumb link, etc. I would suggest create a new issue for that.

Comment (literat) [reply packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx:43]: Ok, let's move on and do it in https://github.com/lmc-eu/spirit-design-system/issues/582. Thanks.

**Review by literat — APPROVED** [review]

**Pull request merged**